force_wfmanager/left_side_pane/tests/test_kpi_specification_model_view.py

```diff
@@ -44,3 +44,12 @@ def test_update_combobox_values(self):
 
     def test_label(self):
         self.assertEqual(self.kpi_specification_mv.label, "KPI")
+
+    def test_name_change(self):
+        self.data_source1.output_slot_info = [OutputSlotInfo(name='T1')]
+        self.kpi_specification_mv.model.name = ''
```

Comment:
I'd add an:
```
with self.assertTraitChanges(self.kpi_specification_mv, 'label', count=1):
```
around this.  It does require adding `UnittestTools` as we talked about yesterday.